core/src/test/java/org/testcontainers/images/builder/DockerignoreTest.java

```diff
@@ -0,0 +1,56 @@
+package org.testcontainers.images.builder;
+
+import static org.rnorth.visibleassertions.VisibleAssertions.assertTrue;
+import static org.rnorth.visibleassertions.VisibleAssertions.fail;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.junit.Test;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.startupcheck.OneShotStartupCheckStrategy;
+
+import com.github.dockerjava.api.exception.DockerClientException;
+
+public class DockerignoreTest {
+    
+    private static final Path INVALID_DOCKERIGNORE_PATH = Paths.get("src/test/resources/dockerfile-build-invalid");
+    
+    @Test
+    public void testInvalidDockerignore() throws Exception {
+        try {
+            new ImageFromDockerfile()
+                .withFileFromPath(".", INVALID_DOCKERIGNORE_PATH)
+                .withDockerfile(INVALID_DOCKERIGNORE_PATH.resolve("Dockerfile"))
+                .get();
+            fail("Should not be able to build an image with an invalid .dockerignore file");
+        } 
+        catch(DockerClientException e) {
```

Comment:
```suggestion
        catch (DockerClientException e) {
```

Reply:
I must say you are quite particular on code formatting @bsideup ;) Perhaps if you want to enforce this strict of formatting conventions, you could add a checkstyle plugin to the project?